rampwf/tests/kits/air_passengers/submissions/starting_kit/submission.py

```diff
@@ -0,0 +1,34 @@
+import os
+import pandas as pd
+from sklearn.preprocessing import FunctionTransformer
+from sklearn.compose import ColumnTransformer
+from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.pipeline import Pipeline
+from sklearn.ensemble import RandomForestRegressor
+
+
+def _merge_weather(X):
+    filepath = os.path.join(os.path.dirname(__file__),
+                            'external_data_mini.csv')
+    X_weather = pd.read_csv(filepath)
+    X_merged = pd.merge(X, X_weather, how='left',
+                        on=['DateOfDeparture', 'Arrival'], sort=False)
+    return X_merged
+
+def get_pipeline():
+    merge_transformer = FunctionTransformer(_merge_weather, validate=False)
+    to_encode = ['Arrival', 'Departure']
+    transformer = ColumnTransformer(transformers=[
+        ('onehotencode', make_pipeline(OneHotEncoder(handle_unknown='ignore')),
+         to_encode),
+        ('drop', 'drop', 'DateOfDeparture')
```

Comment:
Be aware that by default `remainder='drop'` so all unspecified columns will be drop.
So you don't need this.

If you want to keep a set of columns, you need to add a step with a default `FunctionTransformer` which is an identity transformer: `('passthrough`, FunctionTransformer(), passthrough_cols)`

Reply:
Is `('passthrough', FunctionTransformer(), passthrough_cols)` preferred over `('passthrough', 'passthrough', passthrough_cols)`?

Reply:
Uhm I didn't know we can use 'passthrough' but yes. Could we simplify using `make_column_transformer` instead of `ColumnTransformer`.

```python
make_column_transformer(
    (OneHotEncoder(handle_unknown='ignore), categorical_cols),
    ('passthrough', remaining_cols)
)
```

Reply:
Actually it is not all cols that are passed through, one is dropped. But yes I can change to `make_column_transformer`

Reply:
so we can 'drop' the column and pass the parameter `remainder='passthrough'` if you like.